cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True, max_length=255)
+    description = serializers.CharField(required=True)
+    duration = serializers.IntegerField()

```

Comment:
description if not required but you wrote "required" everywhere except duration

Reply:
okey, sorry)))